src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobEntry.cs

```diff
@@ -85,8 +90,14 @@ public async Task CreateAsync(long size, BlobUtilsV12.PageBlobClients pageBlob)
                 },
                 async () =>
                 {
-                    var response = await pageBlob.Default.GetPropertiesAsync();
-                    this.ETag = response.Value.ETag;
+                    try
+                    {
+                        var response = await pageBlob.Default.GetPropertiesAsync();
+                        this.ETag = response.Value.ETag;
+                    }
+                    catch (Azure.RequestFailedException ex) when (BlobUtilsV12.BlobDoesNotExist(ex))
+                    {
+                    }
```

Comment:
So why are we swallowing this exception?

Reply:
I figured it would be good to at least log it, assuming the number of retries isn't all that large.

Reply:
This code gets called if an attempt to create the blob _appears_ to have failed (but may actually have succeeded). In that situation we don't know if the blob does or does not exist. If creation was actually successful, we load the e-tag which then skips creation on the next attempt.

Reply:
After looking at this again, there is actually no reason to swallow the exception because it is not actually expected.

(This code is only called after a previous creation failed with an e-tag exception, so at that point the blob really should exist).

I am therefore removing this try-catch.